src/lib.ts

```diff
@@ -91,7 +90,7 @@ export {
   Network,
   addressUtils,
   cryptoUtils,
-  dateUtils,
+  dateFormatter as dateUtils,
```

Comment:
We already export dateFormatter, maybe we should remove one of these and change on the projects if they use the removed name

Reply:
I agree. Opened #686 to track this removal.